soql-analyzer/src/test/scala/com/socrata/soql/SoQLAnalyzerTest.scala

```diff
@@ -560,4 +560,10 @@ SELECT visits, @x2.zx
         (NoPosition, NoPosition)
     ))
   }
+
+  test("joined table name alias is part of computed function name") {
```

Comment:
I think it is better to remove the word "computed".  We use that word mostly for computed columns.

Reply:
maybe part of expression name